cognite/powerops/resync/models/v2/dms/dayAheadBids/containers.yaml

```diff
@@ -0,0 +1,245 @@
+      properties:
+        - date
+- space: {{model_space}}
+  externalId: BidMethod
+  name: BidMethod
+  usedFor: node
+  properties:
+    name:
+      type:
+        list: false
+        collation: ucs_basic
+        type: text
+      nullable: false
+      autoIncrement: false
+      name: name
+      defaultValue: null
+      description: null
+- space: {{model_space}}
+  externalId: SHOPMultiScenario
+  name: SHOPMultiScenario
+  usedFor: node
+  properties:
+    shopCases:
+      type:
+        list: true
+        collation: ucs_basic
+        type: text
+      nullable: true
+      autoIncrement: false
+      name: shopCases
+      defaultValue: null
+      description: 'external id linked to production data'
+    priceScenarios:
+      type:
+        list: true
+        type: timeseries
+      nullable: true
+      autoIncrement: false
+      name: priceScenarios
+      defaultValue: null
+      description: null
+  constraints:
+    requiredBidMethod:
+      constraintType: requires
+      require:
+        type: container
+        space: {{model_space}}
+        externalId: BidMethod
+- space: {{model_space}}
+  externalId: PriceArea
+  name: PriceArea
+  usedFor: node
+  properties:
+    name:
+      type:
+        list: false
+        collation: ucs_basic
+        type: text
+      nullable: true
+      autoIncrement: false
+      name: name
+      defaultValue: null
+    defaultMethod:
+      type:
+        container:
+          space: {{model_space}}
+          externalId: BidMethod
+        type: direct
+      nullable: true
+      autoIncrement: false
+      name: defaultMethod
+      defaultValue: null
+      description: null
+    timezone:
+      type:
+        list: false
+        collation: ucs_basic
+        type: text
+      nullable: true
+      autoIncrement: false
+      name: timezone
+      defaultValue: null
+    mainScenario:
+      type:
+        list: false
+        type: timeseries
+      nullable: true
+      autoIncrement: false
+      name: timezone
+      defaultValue: null
+    priceScenarios:
+      type:
+        list: true
+        type: timeseries
+      nullable: true
+      autoIncrement: false
+      name: timezone
+      defaultValue: null
+- space: {{model_space}}
+  externalId: BidTable
+  name: BidTable
+  usedFor: node
+  properties:
+    resourceCost:
+      type:
+        list: false
+        collation: ucs_basic
+        type: text
+      nullable: true
+      autoIncrement: false
+      name: resourceCost
+      defaultValue: null
+    table:
+      type:
+        list: false
+        type: sequence
+      nullable: true
+      autoIncrement: false
+      name: table
+      defaultValue: null
+    assetType:
+      type:
+        list: false
+        collation: ucs_basic
+        type: text
+      nullable: true
+      autoIncrement: false
+      name: assetType
+      defaultValue: null
+    assetId:
+      type:
+        list: false
+        collation: ucs_basic
+        type: text
```

Comment:
Should this be int? (or external added to name). Note I think for related events in the Alert we are intending to use internal ids?